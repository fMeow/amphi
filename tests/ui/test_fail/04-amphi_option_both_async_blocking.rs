#![allow(dead_code)]
use amphi::amphi;

#[amphi(async_only, blocking_only)]
mod amphi{
    pub async fn my_fn() -> bool {
        true
    }
}

fn main() {
}
