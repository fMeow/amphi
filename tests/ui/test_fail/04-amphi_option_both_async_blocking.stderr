error: Option `async_only`, `blocking_only` is mutually exclusive
 --> $DIR/04-amphi_option_both_async_blocking.rs:4:21
  |
4 | #[amphi(async_only, blocking_only)]
  |                     ^^^^^^^^^^^^^
